app/src/main/java/com/github/se/eventradar/ui/chat/Chat.kt

```diff
@@ -117,8 +106,6 @@ fun ChatScreenUi(
     }
```

Comment:
**UI bug fix:**
The `LaunchedEffect(key1 = imeBottomPadding)` above is the reason why when we start typing a message and the keyboard opens, all previous messages disappear. Consider removing lines 101-106.